### PR TITLE
367 add links to related resources

### PR DESCRIPTION
**Pull request by ChristophEwertowski**
Resolves https://github.com/hbz/lobid-resources/issues/367

### CODE REVIEW

File: src/test/resources/jsonld/HT002192569 [hunk 1]
```diff
@@ -0,0 +1,172 @@
+{
```
Comment (dr0i) [inline src/test/resources/jsonld/HT002192569]: why adding this new test resource - it seems that the already existing resources do already well testing the "related" data.

File: src/test/resources/output/json/00219/HT002192569.json [hunk 2]
```diff
@@ -0,0 +1,172 @@
+{
```
Comment (dr0i) [inline src/test/resources/output/json/00219/HT002192569.json]: why adding this new test resource - it seems that the already existing resources do already well testing the "related" data.

**Comment by dr0i** [comment]
@acka47 and @ChristophEwertowski:
Everytime a new field is introduced we must also update the `index-config` _if_ the field should be searchable. If not, all is done .
If it _should_ be searchable: 
Should  `related.label` be lower in scoring when evaluate the ranking of the documents? https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-boost.html suggests to not put this in the index mapping but  to put the boost parameter into the query.

**Comment by ChristophEwertowski** [comment]
The resource was needed for an earlier commit but not anymore. Thanks for the tip!
The label doesn't have to be searchable, it's only a nicer hint for humans about the content of the related resource. Important is the hbz-URL/`related.id` where the information also can be looked up.

**Comment by dr0i** [comment]
Ok. Just to be cleared: it's not necessary to query `related.id` ? One would be able to query e.g. "give all resources that are related to http://lobid.org/resources/HT017233564#!" , which would be nice IMO because these information is not reciprocally existing in HT017233564#! , i.e. you can't just have a look at HT017233564#! to list all the related resources.

**Comment by ChristophEwertowski** [comment]
Argh! I asked some collegues about linked resources. Links to parallel resources (print/online, different languages) aren't automatically created in the linked resource. Therefore we should have `related.id` indexed. This would lead to a more ZDB-like search, e.g. you search for an ISSN and also get the parallel edition (but instead with `hbzId`). What do you think @acka47 ?

**Comment by acka47** [comment]
+1 for indexing the `related.id`

**Comment by dr0i** [comment]
+1

**Comment by dr0i** [comment]
Deployed to production, see e.g. http://lobid.org/resources/HT010726584.json. Closed.

**Pull request merged**